README.md

```diff
@@ -805,3 +805,25 @@ on GitHub.
 If you should spot any unclarity or errata within this article, you're
 very welcome to point them out at
 github.com/mamachanko/import-lymph](http://github.com/mamachanko/import-lymph).
+
+
+### Troubleshooting
+
+
+* <a name="install_nfs"> On Ubuntu you can install NFS with</a>
+```bash
+sudo apt-get install nfs-kernel-server
+```
+
+* If you don't have ubuntu/vivid64 box on your pc
+
+```bash
+vagrant box add ubuntu/vivid64 https://cloud-images.ubuntu.com/vagrant/vivid/current/vivid-server-cloudimg-amd64-vagrant-disk1.box
+```
+
+
+* If you get error like
+```bash
+exportfs: /home/oleg/prj/import-lymph does not support NFS export
+```
+That's can be caused that your folder encrypted. So move *import-lymph* inside **unencrypted** folder
```

Comment:
That can be caused by an encrypted folder. Move *import-lymph* inside **unencrypted** folder.